client/app/hosting/database/private-activation/database-private-activation.controller.js

```diff
@@ -1,52 +1,71 @@
-angular.module("App").controller("HostingDatabasePrivateActiveCtrl", ($scope, $rootScope, $stateParams, HostingDatabase, Alerter, Hosting) => {
-    "use strict";
+angular.module("App").controller(
+    "HostingDatabasePrivateActiveCtrl",
+    class HostingDatabasePrivateActiveCtrl {
 
-    $scope.hosting = $scope.currentActionData;
+        constructor ($scope, $rootScope, $stateParams, HostingDatabase, Alerter, PrivateDatabase) {
+            this.$scope = $scope;
+            this.$rootScope = $rootScope;
+            this.$stateParams = $stateParams;
+            this.HostingDatabase = HostingDatabase;
+            this.Alerter = Alerter;
+            this.PrivateDatabase = PrivateDatabase;
 
-    $scope.datas = {
-        versions: []
-    };
-    $scope.loaders = {
-        versions: false
-    };
-    $scope.choice = {
-        ram: $scope.hosting.offerCapabilities.privateDatabases.length === 1 ? $scope.hosting.offerCapabilities.privateDatabases[0] : null,
-        version: null
-    };
+            this.host = this.$scope.currentActionData;
+            this.versions = [];
+            this.loaders = {
+                versions: false
+            };
+            this.choice = {
+                ram: this.host.offerCapabilities.privateDatabases.length === 1 ?
```

Comment:
I don't understand this kind of thing : only have a default choice only if there is only 1 element ? Does it really make sense ? If the user doesn't have a choice, then we shouldn't display a select box.